backend-tests/tests/test_inventory.py

```diff
@@ -277,13 +277,13 @@ def test_ok(self, user):
 
             api_dev = r.json()
             # Expected inventory count per scope:
-            # {"inventory": 3, "identity": 1, "system": 2}
-            assert len(api_dev["attributes"]) == 6
+            # {"inventory": 3, "identity": 1+2, "system": 2} # +2 comes from the id_data see MEN-3637
+            assert len(api_dev["attributes"]) == 8
```

Comment:
I don't understand why the test passed in https://github.com/mendersoftware/deviceauth/pull/351 and then failed in master.

Reply:
I think it is because 351 ran without inventory changes merged in.

Reply:
That could be, but then the inventory changes PR should have failed...

Anyway, for the next time we should use a feature branch for these kind of multi repos changes.